packages/devtools-source-map/src/tests/source-map.js

```diff
@@ -33,7 +33,7 @@ describe("source maps", () => {
     test("source with a url", async () => {
       const urls = await setupBundleFixture("bundle");
       expect(urls).toEqual([
-        "webpack:/webpack/bootstrap 4ef8c7ec7c1df790781e",
+        "webpack:///webpack/bootstrap 4ef8c7ec7c1df790781e",
```

Comment:
what explains this change ?

Reply:
The previous URL here was a bug in source-map, which is now fixed.  See https://github.com/mozilla/source-map/commit/d21b0e1b6e4d6efe7c39cf1607966c417d916be7